test/entrypoint.test.ts

```diff
@@ -74,10 +77,11 @@ describe('EntryPoint', function () {
 
   let accountOwner: Wallet
   const ethersSigner = ethers.provider.getSigner()
-  let account: SimpleAccount
+  let simpleAccount: SimpleAccount
```

Comment:
Why was it renamed? It is not related to the issue, it seems.

Reply:
Because in a test file with 1500 lines and many tests, all accounts are called "account" which is not convenient.